source/agora/node/admin/AdminInterface.d

```diff
@@ -193,8 +193,10 @@ public class AdminInterface : NodeControlAPI
         EncryptionKey encryptionKey = EncryptionKey(
             app,
             Height(height),
-            key
+            key,
+            this.key_pair.address,
         );
+        encryptionKey.signature = this.key_pair.secret.sign(hashFull(encryptionKey)[]);
```

Comment:
```suggestion
        encryptionKey.signature = this.key_pair.secret.sign(encryptionKey);
```

Reply:
We perform the hash in the library itself: https://github.com/bosagora/crypto/blob/ba1eb6b34e8cb680177ea07b7a151d85d5222291/source/agora/crypto/Schnorr.d#L218-L242
But it is hashed with other components.

Reply:
Thanks, I updated.